# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/SessionAccessibility.cpp
# Commit: 3171ba48a786
# Full Hash: 3171ba48a786c4896ff88bb35a83bb6f4c295fd4
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2022-07-07 09:37:57
# Regressor Bug: 1771934
# File Overlap Count: 1
# Description:
#   Bug 1771934 - Lint fix. a=fix. CLOSED TREE
# ==============================================================================

diff -r 4965ca9b7882 -r 3171ba48a786 accessible/android/SessionAccessibility.cpp
--- a/accessible/android/SessionAccessibility.cpp	Wed Jul 06 22:26:08 2022 +0000
+++ b/accessible/android/SessionAccessibility.cpp	Thu Jul 07 02:07:45 2022 +0300
@@ -1081,4 +1081,4 @@
 
     iter.Remove();
   }
-}
\ No newline at end of file
+}
